common.js

```diff
@@ -2716,6 +2716,44 @@ const allSoloCars = {
         '2004': ['ep'],
     },
   },
+  'Scion': {
+    'FR-S (incl. Release Series 1.0 and 2.0)': {
+        'all': ['ssc','stx','csp'],
+    },
+    'FR-S *Limited Prep*': {
+      'all': ['dsp'],
+    },
+    'FR-S (with TRD suspension; incl. Release Series 1.0 and 2.0)': {
+      'all': ['cs'],
+    },
+    'FR-S (without TRD suspension components)': {
+      'all': ['ds'],
+    },
+    'iA': {
+      'all': ['hs','sts'],
+    },
+    'iM': {
+      'all': ['hs','sts'],
+    },
+    'iQ CVT': {
+      'all': ['sts'],
+    },
+    'tC': {
+      'all': ['sts','fsp'],
+    },
+    'tC (incl. Release Series 5.0 2009)': {
```

Comment:
This should probably be merged with the regular `tC` model, just by adding `hs` to the classes array for `tC` and deleting this one.